.github/workflows/cli-coexistence_tests.yaml

```diff
@@ -17,24 +17,14 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout code
-        uses: actions/checkout@v1
+        uses: actions/checkout@v4
```

Comment:
Why this change?
Our other workflows are still on @v1
Should we be migrating all of them?

Reply:
Updated to latest versions since few actions were deprecated .

Yes we should migrate other workflows as well. I will file a JIRA.

Reply:
Thanks!